Workbooks/Communication Services/Overview/Overview.workbook

```diff
@@ -0,0 +1,1031 @@
+                  },
+                  "name": "voice-video-quality-text"
+                },
+                {
+                  "type": 9,
+                  "content": {
+                    "version": "KqlParameterItem/1.0",
+                    "parameters": [
+                      {
+                        "id": "bb9e8d89-0532-49e5-b699-403c33fce006",
+                        "version": "KqlParameterItem/1.0",
+                        "name": "call_quality_media_types",
+                        "label": "Media types",
+                        "type": 2,
+                        "multiSelect": true,
+                        "quote": "'",
+                        "delimiter": ",",
+                        "value": [
+                          "Video",
+                          "Audio",
+                          "VBSS",
+                          "AppSharing"
+                        ],
+                        "typeSettings": {
+                          "additionalResourceOptions": [],
+                          "showDefault": false
+                        },
+                        "jsonData": "[\r\n    {\"value\": \"Audio\", \"label\": \"Audio\", \"selected\":true},\r\n    {\"value\": \"Video\", \"label\": \"Video\", \"selected\":true},\r\n    {\"value\": \"VBSS\", \"label\": \"Screen sharing\", \"selected\":true},\r\n    {\"value\": \"AppSharing\", \"label\": \"App sharing\", \"selected\":true}\r\n]",
+                        "timeContext": {
+                          "durationMs": 86400000
+                        }
+                      },
+                      {
+                        "version": "KqlParameterItem/1.0",
+                        "name": "call_quality_endpoint_types",
+                        "label": "Endpoint types",
+                        "type": 2,
+                        "multiSelect": true,
+                        "quote": "'",
+                        "delimiter": ",",
+                        "typeSettings": {
+                          "additionalResourceOptions": [],
+                          "showDefault": false
+                        },
+                        "jsonData": "[\r\n    {\"value\": \"VoIP\", \"label\": \"VoIP\", \"selected\":true},\r\n    {\"value\": \"PSTN\", \"label\": \"PSTN\", \"selected\":true},\r\n    {\"value\": \"Bot\", \"label\": \"Bot\", \"selected\":true},\r\n    {\"value\": \"Server\", \"label\": \"Server\", \"selected\":true},\r\n    {\"value\": \"Unknown\", \"label\": \"Unknown\", \"selected\":true}\r\n]",
+                        "timeContext": {
+                          "durationMs": 86400000
+                        },
+                        "id": "2666fe6a-842f-46ad-9f75-5e52e8517ff8"
+                      }
+                    ],
+                    "style": "pills",
+                    "queryType": 0,
+                    "resourceType": "microsoft.kusto/clusters"
```

Comment:
maybe this was added by accident? but can delete this

Reply:
Solved this, every resourceType is now Microsoft.Communication/CommunicationServices